src/engraving/compat/midi/compatmidirender.cpp

```diff
@@ -327,11 +327,12 @@ void CompatMidiRender::renderTremolo(Chord* chord, std::vector<NoteEventList>& e
     }
 
     // render tremolo with multiple events
+    int divider = 1 << (tremolo.lines() + chord->durationType().hooks());
+    int t = 1;
+    if (divider != 0) {
+        t = Constants::DIVISION / divider;
+    }
```

Comment:
Now, `t` can still be 0, namely if `divider > Constants::DIVISION`. This will lead to a crash in the calculation of `int tremoloEventsSize`.